CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # [Changelog](https://github.com/yola/opensrs/releases)
 
+## DEV
+
+* Add method to set domain's auto-renew status.
```

Comment:
Since we have 2 interface methods for enabling/disabling, probably we should update this as well?